File: programme/even-or-odd-number/even-or-odd.js
/* Write a program to check whether a number is even or odd

Example:
Input  : 3
Output : Odd Number

*/

const evenOrOdd = (n) => {
  return n % 2 === 0 ? 'Even Number' : 'Odd Number';
};

console.log(evenOrOdd(3));

File: programme/even-or-odd-number/even-or-odd.c
// Write a programme to check a number even or odd
#include <stdio.h>

int main(void)
{
    unsigned long long number;
    printf("Input  : ");
    scanf("%llud", &number);
    printf("Output : %s Number\n", number % 2 == 0 ? "Even" : "Odd");
    return 0;
}
